.pre-commit-config.yaml

```diff
@@ -6,16 +6,20 @@ ci:
     autoupdate_branch: ''
     autoupdate_commit_msg: '[pre-commit.ci] pre-commit autoupdate'
     autoupdate_schedule: quarterly
-    skip: []
+    skip: [bandit]
```

Comment:
Q: I assume this is needed because without it the pre-commit check would currently fail?

Reply:
No, it's because we are running the check via the Bandit GitHub Action separately which then can upload the output SARIF file into GitHub code scanning (https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/uploading-a-sarif-file-to-github). So I figured there's no need to run it twice unnecessarily.

That being said we still have the issue I commented above about the action not allowing us to control the version of bandit at present as it just runs a `pip install bandit[sarif]` command.

Reply:
Asking in upstream here: https://github.com/PyCQA/bandit-action/issues/13